docs/content/users/usage/commands.md

```diff
@@ -680,19 +680,19 @@ Example:
 ddev import-db
 
 # Import the `.tarballs/db.sql` dump to the project database
-ddev import-db --src=.tarballs/db.sql
+ddev import-db --file=.tarballs/db.sql
 
 # Import the compressed `.tarballs/db.sql.gz` dump to the project database
-ddev import-db --src=.tarballs/db.sql.gz
+ddev import-db --file=.tarballs/db.sql.gz
 
-# Import the compressed `.tarballs/db.sql.gz` dump to a `newdb` database
-ddev import-db --target-db=newdb --src=.tarballs/db.sql.gz
+# Import the compressed `.tarballs/db.sql.gz` dump to a `other_db` database
+ddev import-db --database=other_db --file=.tarballs/db.sql.gz
```

Comment:
Maybe `additional_db` if you didn’t feel like `newdb` was working? Seems like a good idea to try and reiterate that it’s referring to the non-primary project database.

```suggestion
# Import the compressed `.tarballs/db.sql.gz` dump to a `additional_db` database
ddev import-db --database=additional_db --file=.tarballs/db.sql.gz
```

Reply:
Done